stripe/res/layout/activity_stripe.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="utf-8"?>
+
+<RelativeLayout
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    >
+
+    <android.support.v7.widget.Toolbar
+
+        android:id="@+id/toolbar_as"
+        android:layout_width="match_parent"
+        android:layout_height="?attr/actionBarSize"
+        android:background="?attr/colorPrimary"
+        android:elevation="@dimen/toolbar_elevation"
+        android:theme="@style/ToolBarStyle"
+        />
+
+    <ScrollView
+        android:layout_below="@+id/toolbar_as"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        >
+
+        <ViewStub
+            android:id="@+id/widget_viewstub_as"
+            android:layout_width="match_parent"
+            android:layout_height="wrap_content"
+            />
+    </ScrollView>
+
+    <ProgressBar
+        android:id="@+id/progress_bar_as"
+        style="@style/Widget.AppCompat.ProgressBar.Horizontal"
+        android:layout_width="match_parent"
```

Comment:
Do you need to make this `layout_below="@id/toolbar_as` so it locates appropriately?